Package.swift

```diff
@@ -19,7 +19,7 @@ let package = Package(
     targets: [
```

Comment:
unfortunately, you'll now need to change the NIO dependency to `from: "2.0.0"` as it won't work with NIO 1

Reply:
:(

Reply:
Is that the case though? Because does NIO 2 and NIO 3 not both use the same methods on `EventLoopPromise` to succeed it? I think just the promise creation changed, no?

Reply:
https://circleci.com/gh/MrLotU/SwiftPrometheus/86?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link ran & succeeded with NIO 1.14.1

Reply:
Nevermind, not true. It only works on NIO 1.x now. I'll push an update :)

Reply:
Can we keep this from 1.0.0 to still work with Vapor 3 💧 until 4 is released?

Maybe we can/should add compiler directives so we could work with both?

Reply:
@Yasumoto / @MrLotU Couldn't you tag version `0.0.0` for NIO1 support and then do `1.0.0` for NIO2?

Reply:
If there is no way to differentiate in code between NIO 1 and 2, than yes, I think that's the way to go.

Reply:
Great save, let's go with a `0.0.0` release train for NIO1.